pages/linux/apt.md

```diff
@@ -0,0 +1,31 @@
+# apt
+
+> Package management utility for Debian based distributions. 
+
+- Synchronize list of packages and versions available. This should be run first, before running subsequent apt commands:
+
+`apt update`
+
+- Search for packages:
+
+`apt search {{package}}`
+
+- Install a new package:
+
+`apt install {{package}}`
+
+- Remove a package (using "purge" instead also removes configuration files):
+
+`apt remove {{package}}`
+
+- Upgrade installed packages to newest available versions:
+
+`apt upgrade`
+
+- Remove no longer needed packages:
+
+`apt autoremove`
+
+- Upgrade installed packages (like "upgrade"), but remove obsolete packages and install additional packages to meet new dependencies:
```

Comment:
Does the "remove obsolete packages" match the behavior of `apt autoremove`? If so, the descriptions should use the same wording. Also, is "install additional packages to meet new dependencies" not true with a plain upgrade?